plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java

```diff
@@ -627,10 +664,35 @@ public abstract Builder maxZoom(@FloatRange(from = MapboxConstants.MINIMUM_ZOOM,
      * Sets the minimum zoom level the map can be displayed at.
      *
      * @param minZoom The new minimum zoom level.
+     * @since 0.5.0
      */
     public abstract Builder minZoom(@FloatRange(from = MapboxConstants.MINIMUM_ZOOM,
       to = MapboxConstants.MAXIMUM_ZOOM) double minZoom);
 
+    /**
+     * Sets the scale factor of the location icon when the map is zoomed in. Based on {@link #maxZoom()}.
+     * Scaling is linear.
```

Comment:
Maybe also worth mentioning the scaling is measured in pixels like the `iconSize` [property docs](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/PropertyFactory.java#L1799) mention